### PR TITLE
updated ethereum, coinbase spellings. removed typo errors

### PR DESCRIPTION
**Pull request by ctrlaltvikas**

### CODE REVIEW

File: app/_locales/index.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
   { "code": "ru", "name": "Russian" },
   { "code": "sl", "name": "Slovenian" },
   { "code": "th", "name": "Thai" },
+  { "code": "tml", "name": "Tamil" },
```
Comment (kumavis) [inline app/_locales/index.json]: we dont have a Tamil locale yet so we can't add it to the index

**Review by kumavis — CHANGES_REQUESTED** [review]

**Comment by kumavis** [comment]
@vikaskyadav thanks for the improvements to Hindi!

 can you add a Tamil translation or remove the Tamil reference from the locales index.json

**Review by ctrlaltvikas — COMMENTED** [review]
removed the Tamil shortcode from index.json. will add that later on.

**Review by kumavis — APPROVED** [review]

**Comment by kumavis** [comment]
@vikaskyadav wow thanks for adding Tamil!

**Comment by kumavis** [comment]
note to merger: please squash and merge

**Comment by vikaskyadav** [comment]
@kumavis . It was supposed to be a PR for gitcoin. We were already discussing in https://github.com/MetaMask/metamask-extension/issues/3549#issuecomment-3795027591
So, do I need to raise the PR since you already have merged?
Let me know I have to Submit the work on gitcoin too

**Comment by kumavis** [comment]
@vikaskyadav sorry for the delay-- merged!

**Pull request merged**